src/values/ObjectValue.js

```diff
@@ -329,7 +329,7 @@ export default class ObjectValue extends ConcreteValue {
       funcName = `get ${name}`;
       if (this.intrinsicName) intrinsicName = `${this.intrinsicName}.${name}`;
     } else if (name instanceof SymbolValue) {
-      funcName = `get [${name.$Description || "?"}]`;
+      funcName = name.$Description instanceof AbstractValue ? name.$Description : `get [${name.$Description || "?"}]`;
```

Comment:
This doesn't seem right, you are not producing the proper string for the AbstractValue case.